WordPress/src/main/java/org/wordpress/android/ui/posts/SelectCategoriesActivity.java

```diff
@@ -118,9 +123,14 @@ public void onRefreshStarted() {
 
         populateCategoryList();
 
-        // Refresh blog list if network is available and activity really starts
-        if (NetworkUtils.isNetworkAvailable(this) && savedInstanceState == null) {
-            refreshCategories();
+        if (NetworkUtils.isNetworkAvailable(this)) {
+            mEmptyView.setText(R.string.empty_list_default);
+            // Refresh blog list if network is available and activity really starts
+            if (savedInstanceState == null) {
+                refreshCategories();
```

Comment:
Just want to confirm one thing, If you open this activity while having no internet, then connect to internet and rotate the screen, this method wont be reached (savedInstanceState wont be null). Is ok?